packages/rfc6287/rfc6287.1.0.4/opam

```diff
@@ -0,0 +1,41 @@
+opam-version: "2.0"
+homepage:     "https://github.com/sg2342/ocaml-rfc6287"
+dev-repo:     "git+https://github.com/sg2342/ocaml-rfc6287.git"
+bug-reports:  "https://github.com/sg2342/ocaml-rfc6287/issues"
+maintainer:   ["Stefan Grundmann <sg2342@googlemail.com>"]
+license:      "BSD2"
+doc: 	      "https://sg2342.github.io/ocaml-rfc6287/doc"
+synopsis: "OCRA (OATH Challenge-Response Algorithm) implementation in OCaml"
+description: """
+OCaml implementation of [RFC6287](http://tools.ietf.org/html/rfc6287) OCRA (OATH Challenge-Response Algorithm)
+incl. [RFC Errata ID: 3729](http://www.rfc-editor.org/errata_search.php?eid=3729)
+"""
+
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs ]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+
+depends: [
+  "ocaml" {>= "4.07.0"}
+  "dune" {>= "1.1"}
+  "mirage-crypto"
+  "mirage-crypto-pk"
+  "mirage-crypto-rng"
+  "cstruct" {>= "3.2.0"}
+  "astring"
+  "hex"
+  "rresult"
+  "ounit" {with-test}
+  "bisect_ppx" {build & >= "1.3.0"}
```

Comment:
Isn't `bisect_ppx` a test dependency instead?

Reply:
Here is the official guideline to avoid having bisect_ppx be a dependency in releases: https://github.com/aantron/bisect_ppx#dune

Reply:
applied the official guideline.